src/visual.ts

```diff
@@ -1503,10 +1505,20 @@ module powerbi.extensibility.visual {
                 .each("end", callback);
         }
         private renderTooltip(selection: UpdateSelection<WordCloudDataPoint>): void {
+            let categorical: WordCloudColumns<DataViewCategoryColumn> = WordCloudColumns.getCategoricalColumns(this.incomingUpdateOptions.dataViews[0]),
+            wordValueFormatter: IValueFormatter = null;
+
+            if (categorical.Values && categorical.Values[0]) {
+                wordValueFormatter = ValueFormatter.create({
+                    format: ValueFormatter.getFormatStringByColumn(categorical.Values[0].source)
```

Comment:
I think it'd be better to specify [culture selector](https://github.com/Microsoft/powerbi-visuals-utils-formattingutils/blob/master/lib/index.d.ts#L749) as well

Reply:
Is there examples how to use it? I don't see that another formatters in visual use it.